.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+# Java Gradle CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+#
+version: 2
+jobs:
+  build:
+    branches:
+      only:
+        - master
+        - /feature\/ci.*/
+    docker:
+      - image: circleci/node:10-browsers-legacy
+        environment:
+          MAX_HEAP_SIZE: 2048m
+          HEAP_NEWSIZE: 512m
+
+    working_directory: ~/web-client-sdk-base
```

Comment:
Is this typo ? why `web` ? Do I misunderstand something ?

Reply:
Owh, i'm sorry i got it mixed up after working with many different repo, i'll change it to `js-sdk-base`. Is that okay?

Reply:
I think anything is fine but `javascript-sdk-base` sounds better than `js-sdk-base` if we use something similar.